### ISSUE TITLE
Constructed queries should have :upperDashboardTime: by default

### ISSUE THREAD

**Issue by Tabinol** [influxdata/chronograf#2760]
When I take "Custom date Picker", the begin date works but the end date is ignored, always "now" no matter what I enter.

The request is for example:

`SELECT mean("usage_user") AS "usage_user" FROM "telegraf"."autogen"."cpu" WHERE time > :dashboardTime: AND "host"=:host-name: GROUP BY :interval: FILL(null)`

But, when I use the default graphics in "Host Name" tab, it is working with no issue.

I saw this problem in Chronograf 1.3.10-0 and Chronograf 1.4.0-1. I use Docker version.

Firefox and Google Chrome on Windows 10 give the same result. There is no problem in "Host list"

![gd2hh](https://user-images.githubusercontent.com/8842894/35869122-5925cf9a-0b2c-11e8-8121-f1965837301c.png)

**Comment by goller** [comment]
@Tabinol thanks for writing in.   I just looked into the answer to this and you'll need to change your queries to look like this:

```
SELECT mean("usage_user") AS "usage_user" FROM "telegraf"."autogen"."cpu" WHERE time > :dashboardTime: AND time < :upperDashboardTime: AND "host"=:host-name: GROUP BY :interval: FILL(null)
```

However, that's particularly hard to discover.  Would you try that query out and see if it works for you?

Also, let's keep this issue open, but, would you change the title to be something like, "constructed queries should have :upperDashboardTime: by default"

**Comment by Tabinol** [comment]
Your solution is working. Thank you.

Yes I just change the ticket title because it is not clear and it would be nice if the default request would support the upper Dashboard Time.

**Comment by regel** [comment]
+1: It is useful. Thank you @Tabinol you've made my day!

**Comment by russorat** [comment]
i believe we addressed this in 1.6 https://github.com/influxdata/chronograf/issues/3517

You can still use `:upperDashboardTime:` if you need to build complex queries, but just using `:dashboardTime:` in your queries will work when using custom date selectors in the past.
